CHANGELOG.md

```diff
@@ -4,41 +4,65 @@
 `psql` with the `-X` flag to prevent any `.psqlrc` commands from
 accidentally triggering the load of a previous DB version.**
 
-## Unreleased
+## 2.9.0
+This release adds major new features since the 2.8.1 release.
+We deem it moderate priority for upgrading.
+
+This release includes these noteworthy features:
+* Hierarchical Continuous Aggregates (aka Continuous Aggregate on top of another Continuous Aggregate)
+* Improve `time_bucket_gapfill` function to allow specifying the timezone to bucket
+* Introduce fixed schedules for background jobs and the ability to check job errors.
+* Use `alter_data_node()` to change the data node configuration. This function introduces the option to configure the availability of the data node.
 
-This version adds time_bucket_gapfill function that allows
-specifying the timezone to bucket for. Unfortunately this introduces
-an ambiguity with some previous call variations when an untyped
-start/finish argument is passed to time_bucket_gapfill. Some queries
-might need to be adjusted and either explicitly name the positional
-argument or resolve the type ambiguity by casting to the intended type.
+This release also includes several bug fixes.
 
 **Features**
+* #4476 Batch rows on access node for distributed COPY
+* #4567 Exponentially backoff when out of background workers
```

Comment:
These two don't look like feature, it's an implementation detail. But anyway, I see that this is a copy of changelog from main so maybe we can leave it as is.

Reply:
Okay